src/SAWScript/CrucibleBuiltins.hs

```diff
@@ -155,6 +182,31 @@ ppAbortedResult _ (Crucible.AbortedBranch _ _ _) =
 ppAbortedResult _ (Crucible.AbortedExit ec) =
   text "Branch exited:" <+> text (show ec)
 
+matchingStatics :: L.Symbol -> L.Symbol -> Bool
```

Comment:
A comment here might be nice, I can imagine being confused about the purpose of this function

Reply:
Done!